Sources/AWSLambdaRuntimeCore/LambdaContext.swift

```diff
@@ -32,13 +33,17 @@ extension Lambda {
         /// - note: The `EventLoop` is shared with the Lambda runtime engine and should be handled with extra care.
         ///         Most importantly the `EventLoop` must never be blocked.
         public let eventLoop: EventLoop
+        
+        /// `ServiceLifecycle` to register services with
+        public let serviceLifecycle: ServiceLifecycle
```

Comment:
maybe simplify name to "lifecycle"?

Reply:
Thank you

Reply:
Not sure if just `lifecycle` might add confusion because of the `Lambda.Lifecycle`